ui/packages/atlasmap/src/impl/utils/document.ts

```diff
@@ -199,6 +199,16 @@ export async function getCustomClassNameOptions(): Promise<string[]> {
   });
 }
 
+/**
```

Comment:
The `getCustomClassNameOptions()` above should be removed and the `DocumentManagementService.getLibraryClassNames()` should be used directly, as it doesn't add anything. Then this should be also in the `DocumentManagementService`, next to the `getLibraryClassNames()`.